src/Files.App.Storage/WindowsStorage/WindowsStorageService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Files.Sdk.Storage;
+using Files.Sdk.Storage.LocatableStorage;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+
+namespace Files.App.Storage.WindowsStorage
+{
+	/// <inheritdoc cref="IStorageService"/>
+	public sealed class WindowsStorageService : IStorageService
+	{
+		/// <inheritdoc/>
+		public Task<bool> IsAccessibleAsync(CancellationToken cancellationToken = default)
+		{
+			return Task.FromResult(true);
+		}
+
+		/// <inheritdoc/>
+		public async Task<bool> FileExistsAsync(string path, CancellationToken cancellationToken = default)
+		{
+			try
+			{
+				_ = await GetFileFromPathAsync(path, cancellationToken);
+				return true;
+			}
+			catch (Exception)
+			{
+				return false;
+			}
+		}
+
+		/// <inheritdoc/>
+		public async Task<bool> DirectoryExistsAsync(string path, CancellationToken cancellationToken = default)
+		{
+			try
+			{
+				_ = await GetFolderFromPathAsync(path, cancellationToken);
+				return true;
+			}
+			catch (Exception)
+			{
+				return false;
+			}
+		}
+
+		/// <inheritdoc/>
+		public async Task<ILocatableFolder> GetFolderFromPathAsync(string path, CancellationToken cancellationToken = default)
+		{
+			var folderTask = StorageFolder.GetFolderFromPathAsync(path).AsTask(cancellationToken);
+			var folder = await folderTask;
```

Comment:
Cannot check on an IDE, but could it be made directly as follow instead of a two liner?
```
var folder = await StorageFolder.GetFolderFromPathAsync(path).AsTask(cancellationToken);
```